src/Context/AuthContext.js

```diff
@@ -6,10 +6,13 @@ const AuthProvider=({children})=>{
     const [isLogedIn,setLogedIn]=useState(userLoggedIn)
     const [userDetail,setUserdetail]=useState({token:"",user: {}})
     const token=localStorage.getItem("token")
+    console.log(token,"gettoken")
     const user=JSON.parse(localStorage.getItem("user"))
-
+    console.log(token ,user,"now")
```

Comment:
remove console logs

Reply:
> remove console logs

thx @Tanjum786  for review I will remove console